librz/analysis/type_pdb.c

```diff
@@ -28,237 +49,634 @@ static char *create_type_name_from_offset(ut64 offset) {
 	return str;
 }
 
+static RzType *parse_type_array(const RzTypeDB *typedb, STypeInfo *type) {
+	SLF_ARRAY *lf_array = type->type_info;
+	RzType *typ = RZ_NEW0(RzType);
+	if (!typ) {
+		return NULL;
+	}
+	typ->kind = RZ_TYPE_KIND_ARRAY;
+	SType *element = rz_bin_pdb_stype_by_index(lf_array->element_type);
+	if (!element) {
+		rz_type_free(typ);
+		return NULL;
+	}
+	RzType *element_type = parse_type(typedb, element);
+	if (!element_type) {
+		rz_type_free(typ);
```

Comment:
Shouldn't we also free `element` here?

Reply:
The element will be freed when the parser was finished.